.github/workflows/typing.yaml

```diff
@@ -49,4 +49,4 @@ jobs:
       - name: 🏗 Install dependencies
         run: poetry install --no-interaction
       - name: 🚀 Run mypy
-        run: poetry run mypy omnikinverter tests
+        run: poetry run mypy **/*.py
```

Comment:
Just keep it to only the package and tests

Reply:
Let's discuss this specific bit in #160 (this PR is based on that one so that `mypy` is included when I run `pre-commit` before pushing :wink:).

In any case, this more closely resembles how `.pre-commit-config.yaml` is set up, but I've changed it back and only added `test_output.py` on the end, so that all files are tested accordingly.